### PR TITLE
fix draw wrong position while playing backward seeking

### PR DESCRIPTION
**Pull request by candyan**
while audio is playing, i seek to backward. the cursor is still in its original position. until playing to the original position. the cursor is return to normal.

### CODE REVIEW

File: src/wavesurfer.js [hunk 1]
```diff
@@ -183,16 +183,17 @@ var WaveSurfer = {
         this.fireEvent('interaction', this.seekTo.bind(this, progress));
 
         var paused = this.backend.isPaused();
+        // avoid draw wrong positon while playing backward seeking
```
Comment (thijstriemstra) [inline src/wavesurfer.js]: "positon" should be "position".

Comment (candyan) [inline src/wavesurfer.js]: thx, my mistake

**Comment by katspaugh** [comment]
I don't quite understand the issue that is being solved here. I don't see any problem with seeking backwards on the homepage.

**Comment by thijstriemstra** [comment]
@candyan can you explain this PR in more detail?

**Comment by qwertycopter** [comment]
I can confirm this issue exists and candyan's edits fix it.
When playing audio and clicking back to a previous part of the waveform sometimes the cursor does not move back (although the correct audio will play)
I'm using the MediaElement backend and very long ogg files which might contribute to the problem (the browser does not download the whole ogg file but makes several 206 partial content requests)

**Comment by katspaugh** [comment]
Ah, so it's only for the MediaElement. Makes sense. I'll merge then.

**Pull request merged**